tests.toml

```diff
@@ -0,0 +1,35 @@
+test_format = 1.0
+
+[default]
+
+    # -------------------------------
+    # Default args to use for install
+    # -------------------------------
+    
+    args.export = "libreoffice"
+    args.mypads = 1
+    args.useldap = 1
+
+    # -------------------------------
+    # Commits to test upgrade from
+    # -------------------------------
+
+    test_upgrade_from.4d613658.name = "Upgrade from 1.8.17"
+
+
+# This is an additional test suite
+[sans_MyPads]
+    args.export = "libreoffice"
+    args.mypads = 0
+    args.useldap = 0
+
+
+[Abiword]
+    args.export = "abiword"
+    args.mypads = 1
+    args.useldap = 1
+
+[sans_export]
+    args.export = "none"
+    args.mypads = 1
+    args.useldap = 0
```

Comment:
```suggestion
# This is an additional test suite
[without_mypads]
    args.export = "libreoffice"
    args.mypads = 0
    args.useldap = 0
    only = ["install.root"]

[with_abiword]
    args.export = "abiword"
    args.mypads = 1
    args.useldap = 1
    only = ["install.root"]

[without_export]
    args.export = "none"
    args.mypads = 1
    args.useldap = 0
    only = ["install.root"]
```

I suggest adding `only = ["install.root"]`, because enabling mypads or export = none/libreoffice/abiword should be independent of the install, upgrade, backup, etc. operations ... Otherwise right now we end up with like 33 tests which are super long and never gonna go past the CI max time which is 3 hours for all tests :/